### ISSUE TITLE
Wrong step's statuses in allure report

### ISSUE THREAD

**Issue by voronsp** [saksmt/ktest#41]
Hi!
I found, that checking for exception will set status in broken or fail with condition e==null instead e!=null, so i have broken steps for good ones:
```
if (e == null) {
            allure.updateStep { it
                .withStatus(ResultsUtils.getStatus(e).orElse(Status.BROKEN))
                .withStatusDetails(ResultsUtils.getStatusDetails(e).orElse(null))
                .let { if (newName == null) it else it.withName(newName) }
            }
        } else {
            allure.updateStep { it.withStatus(Status.PASSED).let { if (newName == null) it else it.withName(newName) } }
        }
```

**Comment by saksmt** [comment]
Thanks for your contribution! Will be fixed by today evening

**Comment by saksmt** [comment]
Grab the fix in v1.0.0-rc2 😄
